### ISSUE TITLE
Blender's vertex color attribute names don't work in Smash

### ISSUE THREAD

**Issue by ScanMountGoat** [ssbucarlos/smash-ultimate-blender#83]
This can cause various errors in applications like Cross Mod and StudioSB. I don't believe adding unneeded attributes to a mesh causes any issues in game. There are different ways to approach this. 
- automatically rename the vertex color set to "colorSet1" on export
- prevent export and give an error message to rename the attribute

My personal preference is to raise an error since it's a pretty easy fix for the user if they have specific instructions on what they need to change.

**Comment by ScanMountGoat** [comment]
StudioSB (and Cross Mod to some extent) are fundamentally broken in how they process mesh attribute data, so maintaining compatibility with these programs shouldn't be a huge concern.

**Comment by ssbucarlos** [comment]
Yeah i would prefer to prevent the export as well, with instructions on how to change. a similar issue exists for uvMaps.
